devops/providers/azure-devops/templates/app_service_maven/tasks/app-service-elastic-setup.yml

```diff
@@ -0,0 +1,35 @@
+#####################
```

Comment:
Similar comment as to what @thephen made but this might not make sense for the back-port

Reply:
Definitely stripped this out along with it's variable groups. Thanks for pointing this out.